grails-app/services/org/pih/warehouse/inventory/OutboundStockMovementService.groovy

```diff
@@ -39,6 +39,7 @@ class OutboundStockMovementService {
         Date createdAfter = params.createdAfter ? Date.parse("MM/dd/yyyy", params.createdAfter) : null
         Date createdBefore = params.createdBefore ? Date.parse("MM/dd/yyyy", params.createdBefore) : null
         List<ShipmentType> shipmentTypes = params.list("shipmentType") ? params.list("shipmentType").collect{ ShipmentType.read(it) } : null
+        Boolean isApprovalRequired = stockMovement?.origin?.isApprovalRequired()
```

Comment:
not a change request: since it is a transient, you can access it through the property like: `origin.approvalRequired`